boomer/boomer.go

```diff
@@ -17,6 +17,7 @@ package boomer
 
 import (
 	"crypto/tls"
+	"golang.org/x/net/http2"
```

Comment:
Could you add a line after standard library imports and add the http2 package afterwards?
````
import (
    "crypto/tls"
    ...
    "time"

    "golang.org/x/net/http2"
)
````